package/post-install.sh

```diff
@@ -8,15 +8,81 @@ INI_FILE_NAME='99-ddtrace.ini'
 
 PATH="${PATH}:/usr/local/bin"
 
+function println(){
+    echo -e '###' "$@"
+}
+
+function append_configuration_to_file() {
+    tee -a "$@" <<EOF
+## Autogenerated by the DataDog post-install.sh script
+
+[datadog]
+extension=${EXTENSION_FILE_PATH}
+
+## end of autogenerated part
+EOF
+}
+
+function install_conf_d_file() {
+    INI_FILE_PATH="${EXTENSION_CFG_DIR}/$INI_FILE_NAME"
+
+    println "Creating ddtrace.ini"
+    println "\n"
+
+    append_configuration_to_file "${INI_FILE_PATH}"
+
+    println "ddtrace.ini created"
+    println
+
+    PHP_DDTRACE_INI="$PHP_CFG_DIR/$INI_FILE_NAME"
+
+    println "Linking ddtrace.ini to ${PHP_DDTRACE_INI}"
+    test -f "${PHP_DDTRACE_INI}" && rm "${PHP_DDTRACE_INI}"
+    ln -s "$INI_FILE_PATH" "${PHP_DDTRACE_INI}"
+}
+
+function fail_print_and_exit() {
+    println 'Failed enabling ddtrace extension'
+    println
+    println "The extension has been installed and couldn't be enabled"
+    println "Try adding the extension manually to your PHP - php.ini - configuration file"
+    println "e.g. by adding following line: "
+    println
+    println "    extension=${EXTENSION_FILE_PATH}"
+    println
+    println "Note that your PHP API version must match the extension's API version"
+    println "PHP API version can be found using following command"
+    println
+    println "    php -i | grep 'PHP API'"
+    println
+
+    exit 0 # exit - but do not fail the installtion
```

Comment:
the whole meaning of this is that we still want to succeed so user can go and add it manually. is it my interpretation correct?

Reply:
Yes - exactly. If someone has a very custom setup it still should be possible to install the package correctly.